packages/libs/react-components/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,30 @@
+import { Heading } from '../Typography';
+
+import { StyledCard } from './styles';
+
+import { VariantProps } from '@stitches/react';
+import React, { FC } from 'react';
+
+export interface ICardProps {
+  children: React.ReactNode;
+  color?: VariantProps<typeof StyledCard>['color'];
+  expand?: boolean;
```

Comment:
Since expand is a Variant, I would recommend using the VariantProp typing for the sake of consistency

Reply:
👀